bitcoin/src/amount.rs

```diff
@@ -1530,6 +1530,18 @@ mod tests {
 
     use super::*;
 
+    #[test]
+    fn from_str_basic() {
+        assert!(Amount::from_str("1 BTC").is_ok());      // 100,000,000 sats
+        assert!(Amount::from_str("1 mBTC").is_ok());     // 100,000 sats
+        assert!(Amount::from_str("1 uBTC").is_ok());     // 100 sats
+        assert!(Amount::from_str("10 nBTC").is_ok());    // 1 sat
+        assert!(Amount::from_str("10000 pBTC").is_ok()); // 1 sat
+        assert!(Amount::from_str("1 bit").is_ok());      // 100 sats
+        assert!(Amount::from_str("1 sat").is_ok());      // 1 sat
+        assert!(Amount::from_str("1000 msats").is_ok()); // 1 sat
+    }
```

Comment:
Why not also compare the value while we're at it?

Reply:
Yeah, I suppose so. The number of sats is there in the comment anyways so might as well use `Amount::from_sat` or what ever it is. I'll re-spin, thanks.